bench/index.html

```diff
@@ -12,16 +12,36 @@
 
 <body>
     <div id="benchmarks"></div>
-    <script src="/bench/benchmarks_generated-v0.43.0.js"></script>
     <script>
-        // Unless the URL contains a no-master query parameter, include the
-        // current master-branch build of benchmarks.js for comparison
-        if (!/no-master/.test(window.location.search)) {
-            document.write(`<script src="https://s3.amazonaws.com/mapbox-gl-js/master/benchmarks.js"><` + `/script>`);
-        }
+        const params = {};
+        (window.location.search || '?').slice(1)
```

Comment:
How about using [`URLSearchParams`](https://stackoverflow.com/questions/901115/how-can-i-get-query-string-values-in-javascript/901144#12151322)?